scss/components/_table-styles.scss

```diff
@@ -253,6 +253,84 @@ h3 + .simple-table {
   td:first-child {
     border-right: 1px solid $gray-medium;
     padding-left: u(4rem);
-    font-style: italic;
+     font-style: italic;
   }
 }
+
+// For custom_table_block in streamfactory
+
+.custom-table {
+  box-shadow: -8px 0px 9px -2px $gray-medium inset;
+  overflow-x: scroll;
+  display: block;
+  margin-bottom: u(1rem);
+
+  table {
+    @extend .simple-table;
+    width: auto;
+
+    th {
+      @extend .simple-table__header;
+      table-layout: auto;
+      padding: u(1rem);
+      text-align: left;
+      border-right: 1px solid $gray-medium;
+    }
+
+    tr {
+      @extend .simple-table__row;
+      th:first-child {
+        border-bottom: 1px solid $neutral;
+      }
+    }
+
+    td {
+      @extend .simple-table__cell;
+      border-right: 1px solid $gray-medium;
+      text-align: right;
+      padding: u(1rem) !important;
+    }
+
+    td:last-child,
+    thead th:last-child {
+      border-right: none;
+    }
+
+    tr td:first-child {
+      font-weight: bold;
+      text-align: left;
+    }
+
+    thead tr th:first-child {
+      border-bottom: 1px solid $primary;
+    }
+  }
+
+  @include media($lg) {
+    box-shadow: none;
+  }
+
+  @include media(min-width 640px max-width 755px 12) {
```

Comment:
**"I think we can simplify these and just not worry about hiding it at in-between sizes (just hide it on the large size). What do you think?"**
I agree, I figured that might be a suggestion--which is better because then we are sticking to "mobile-first" and not using any max-width media queries. 
Also those arbitrary breakpoints  I came up with for where to hide/show probably will not be exactly the same across differing environments.